docs/dev-tools/zombienet/overview.md

```diff
@@ -235,417 +243,319 @@ Zombienet provides a CLI that allows interaction with the tool. The CLI can rece
-    | `prometheus_port`            | Number           | Prometheus port to use                                                                           | -               |
-    | `prometheus_prefix`          | String           | Customizing the metric's prefix for the specific node                                            | `substrate`     |
-    | `keystore_key_types`         | String           | Defines which keystore keys should be created                                                    | -               |
-
-    So, for example, the following configuration file defines a minimal example for the relay chain, including the `nodes` key:
-
-    === "relaychain-example-nodes.toml"
-        ```toml
-        [relaychain]
-        default_command = "polkadot"
-        default_image = "polkadot-debug:master"
-        chain = "rococo-local"
-        chain_spec_path = "/path/to/chain-spec.json"
-        default_args = ["--chain", "rococo-local"]
-
-        [[relaychain.nodes]]
-        name = "alice"
-        validator = true
-        balance = 1000000000000
-
-        [[relaychain.nodes]]
-        name = "bob"
-        validator = true
-        balance = 1000000000000
-        ...
-        ```
+You can use the `relaychain` keyword to define further parameters for the relay chain at start-up. The available keys are:
+
+- `default_command` ++"string"++ - the default command to run. Defaults to `polkadot`
+- `chain` ++"string"++ - the chain name
+- `chain_spec_path` ++"string"++ - path to the chain spec file. Should be the plain version to allow customizations
+- `chain_spec_command` ++"string"++ - command to generate the chain spec. It can't be used in combination with `chain_spec_path`
+- `default_args` ++"string[]"++ - an array of arguments to use as default to pass to the command
+- `default_substrate_cli_args_version` ++"enum"++ - set the Substrate CLI args version
+- `default_overrides` ++"Override object[]"++ - an array of overrides to upload to the nodes
+- `default_resources` ++"Resources object"++ - represents the resources limits/reservations needed by the nodes by default. Only available on Kubernetes
+- `default_prometheus_prefix` ++"string"++ - a parameter for customizing the metric's prefix. Defaults to `substrate`
+- `random_nominators_count` ++"number"++ - if set and the stacking pallet is enabled, Zombienet will generate the input quantity of nominators and inject them into the genesis
+- `max_nominations` ++"number"++ - the max number of nominations allowed by a nominator. Should match the value set in the rumtime. Defaults to `24`
+
+### Node Configuration
+
+There is one specific key capable of receiving more subkeys: the `nodes` key. This key is used to define further parameters for the nodes. The available keys:
+
+- `name` ++"string"++ - name of the node. Any whitespace will be replaced with a dash (e.g., 'new alice' -> 'new-alice')
```

Comment:
```suggestion
- `name` ++"string"++ - name of the node. Any whitespace will be replaced with a dash (e.g., `new alice` will be converted to `new-alice`)
```

Since this is referring to code, let's use back ticks instead of single quotes